.travis.yml

```diff
@@ -1,18 +1,22 @@
----
 sudo: required
 dist: bionic
 language: python
-python: "3.7.4"
+python: 3.7.4
 
 install:
   - pip install -r requirements.txt
 
 script:
   - ansible-lint playbooks/*.yml
   - yamllint **/*.yml
-  - sudo apt remove --purge postgresql-9.* postgresql-10
-  - sudo rm -rf /etc/postgresql/9.4/ /var/lib/postgresql/9.4/
-  - sudo apt-get update
   - ansible-galaxy install -r requirements.yml
-  - ansible-playbook playbooks/sys_admins.yml --limit=travis_ci
-  - ansible-playbook playbooks/provision.yml --limit=travis_ci
+  - ANSIBLE_HOST_KEY_CHECKING=False ansible-playbook playbooks/sys_admins.yml --limit=ci_server
+  - ANSIBLE_HOST_KEY_CHECKING=False ansible-playbook playbooks/provision.yml --limit=ci_server
+    -vvv
+
+before_install:
```

Comment:
this is what the Travis CLI adds when you run `travis encrypt-file travis --add`